calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,574 @@
+| Threshold value recommendation        | Compare the value for **Total Connections Accepted** and **Client Connections Actively Streaming**. The fluctuation of these values should be in-sync with each other if Accepted Connections are turning into Actively Streamed connections. If there is a discrepancy , you should investigate. **Note**: As always, it is recommended to baseline the relationship between these two metrics to have a sense of what is normal. It is also worth noting that in smaller clusters, it is normal for Typha to be unbalanced. Typha can handle hundreds of connections so it is of no concern if all nodes in a 10-node cluster (for example) connect to the same Typha. |
+| Threshold breach symptoms             | Felix is not getting updates from Typha. {{prodname}} network policies are out-of-sync. |
+| Threshold breach recommendations      | Check Typha and Felix logs, and rollout restart Typha(s) if needed. |
+| Priority level                        | Recommended.                                                 |
+
+### Rebalanced client connections
+
+| Rebalanced client connections    |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>rate(typha_connections_dropped{$_rate_interval})</code> |
+| Example value                    | <code>{endpoint="metrics-port", instance="10.0.1.20:9093", job="typha-metrics-svc", namespace="calico-system", pod="calico-typha-6c6cc9fcf7-csbdl", service="typha-metrics-svc"}</code> |
+| Explanation                      | Number of client connections dropped to rebalance and share the load across different Typhas. |
+| Threshold value recommendation   | It is normal to see this value increasing sometimes. Investigate if connection dropped counters is rising constantly. If all Typhas are dropping connections because all Typhas believe they have too much load, this also warrants investigation. |
+| Threshold breach symptoms        | Dropping connections is rate limited so it should not affect the cluster as a whole. Typha clients, like Felix, will get dropped every sometimes (but not constantly), and could result in periodic delays to policy updates. |
+| Threshold breach recommendations | Ensure that the Kubernetes nodes have enough resources.      |
+| Priority level                   | Optional.                                                    |
+
+### 99 percentile client fall-behind
+
+| 99 percentile client fall-behind |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | <code>max by (instance) (typha_client_latency_secs{quantile='0.99'})</code> |
+| Example value | <code>{instance="10.0.1.20:9093"} NaN</code><br /><code>{instance="10.0.1.31:9093"} NaN</code> |
```

Comment:
NaN means "not a number", suggests the query failed, possibly the cluster was so quiet that there have been no non-blocking reads and the calculation fails.  Annoying that prometheus treats percentiles that way.

You could just put a number in there like `0.1234`

Reply:
Changed NaN to 0.1234